### PR TITLE
Fixed system to use arable specified spectrum in new recipe

### PR DESCRIPTION
**Pull request by srmoore**
Small change to generate a light spectrum from the arable data for use in the recipe.

### CODE REVIEW

File: cc/recipe/recipe_data.py [hunk 1]
```diff
@@ -181,12 +181,32 @@ def create_recipe(self,
...
+                                "light_band3_w_m2",
+                                "light_band4_w_m2",
+                                "light_band5_w_m2",
+                                "light_band6_w_m2",
+                                "light_band7_w_m2"]
+            for band_name in light_band_names:
+                light_bands.append(w[band_name])
+            total_light_val = sum(light_bands)
+            light_bands_pct = list(map(lambda x: (x / total_light_val) * 100.0, light_bands))
+            print(type(light_bands_pct))
```
Comment (rbaynes) [inline cc/recipe/recipe_data.py]: can remove this line

File: cc/recipe/recipe_data.py [hunk 2]
```diff
@@ -181,12 +181,32 @@ def create_recipe(self,
...
+                               "620-690": light_bands_pct[3],
+                               "691-740": light_bands_pct[4],
+                               "780-900": light_bands_pct[5],
+                               "930-960": light_bands_pct[6]}
             logging.info(f'{name} {temp:4.2f} {RH:6.2f} {PAR:7.2f}')
 
             # Add a named environment 
             template_recipe_dict["environments"][name] = {
                 "name": name,
-                "light_spectrum_nm_percent": PFC_sun_spectrum,
```
Comment (rbaynes) [inline cc/recipe/recipe_data.py]: can remove unused PFC_sun_spectrum dict

File: cc/recipe/recipe_data.py (hunk 1 above)
Comment (srmoore) [reply cc/recipe/recipe_data.py]: Good catch

File: cc/recipe/recipe_data.py (hunk 2 above)
Comment (srmoore) [reply cc/recipe/recipe_data.py]: Looks like it is defined in the manual set one anyway, so no need to have it up above. Cleaned up a little.

**Pull request merged**